sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs

```diff
@@ -65,9 +65,9 @@ public bool IsClosed
         internal ServiceBusEventSource Logger { get; set; } = ServiceBusEventSource.Log;
 
         /// <summary>
-        /// In the case of a via-sender, the message is sent to <see cref="EntityPath"/> via <see cref="ViaEntityPath"/>; null otherwise.
+        /// In the case of a via-sender, the message is sent to <see cref="EntityPath"/> via <see cref="TransactionEntityPath"/>; null otherwise.
```

Comment:
I'm not sure that this helps to explain the concept to those that may not be familiar.   In particular, the term `via-sender` may benefit from some more words to define the concept.

Reply:
linked to the options property which has the full explanation.